### PR TITLE
Initial commit of FAudio package

### PR DESCRIPTION
**Pull request by ZachBacon**
This provides an XAudio2 like api using SDL2 and ffmpeg (For wma decoding) a look over would be nice to make sure files are properly placed for the cmake config files.

### CODE REVIEW

**Comment by Alexpux** [comment]
@ZachBacon I will not accept git packages for anty reason. I'm trying to minimize them instead

**Comment by ZachBacon** [comment]
Fair enough, may I ask that this keeps open until a release is done by the faudio team?

**Comment by ZachBacon** [comment]
@AlexPux Just got word, beginning January there will be tagged releases of faudio, once the first tag is in, I'll update the pkgbuild

**Comment by ZachBacon** [comment]
@Alexpux First official FAudio Release has finally happened, updated PKGBUILD. It should be alright, but I'll leave final say to you.

File: mingw-w64-faudio/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+pkgver="19.01"
+pkgrel=1
+pkgdesc="FAudio - Accuracy-focused XAudio reimplementation for open platforms (git) (mingw-w64)"
+arch=('any')
+url="https://github.com/FNA-XNA/FAudio"
+license=('custom')
+makedepends=("${MINGW_PACKAGE_PREFIX}-cmake"
+             "${MINGW_PACKAGE_PREFIX}-ffmpeg"
+             "${MINGW_PACKAGE_PREFIX}-SDL2"
+             'git')
```
[the remark this answers is not in the document]
Comment (ZachBacon) [reply mingw-w64-faudio/PKGBUILD]: Yeah missed that one as well as the others. When I can tonight I'll send the changes.

**Comment by ZachBacon** [comment]
@Alexpux I know you're busy and all, but a quick review if you please.

File: mingw-w64-faudio/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+pkgrel=1
+pkgdesc="FAudio - Accuracy-focused XAudio reimplementation for open platforms (mingw-w64)"
+arch=('any')
+url="https://github.com/FNA-XNA/FAudio"
+license=('custom')
+makedepends=("${MINGW_PACKAGE_PREFIX}-cmake"
+             "${MINGW_PACKAGE_PREFIX}-ffmpeg"
+             "${MINGW_PACKAGE_PREFIX}-SDL2")
+options=('strip' 'staticlibs')
+source=("https://github.com/FNA-XNA/FAudio/archive/${pkgver}.tar.gz")
```
Comment (Alexpux) [inline mingw-w64-faudio/PKGBUILD]: download into named archive like:
````${_realname}-${pkgver}.tar.gz::"https://github.com/FNA-XNA/FAudio/archive/${pkgver}.tar.gz"````

Comment (ZachBacon) [reply mingw-w64-faudio/PKGBUILD]: I'll do that, when I get home. I want this merged in as soon as possible if I can.

**Pull request merged**